server/routes/varyingLicences/handlers/viewActiveLicence.test.ts

```diff
@@ -73,5 +121,93 @@ describe('Route Handlers - Vary Licence - View active licence', () => {
 
       expect(res.redirect).toHaveBeenCalledWith(`/licence/vary/id/${licence.id}/timeline`)
     })
+
+    it('should display expired section if licence type is AP_PSS, is in pss period, isActivatedInPssPeriod with additional conditions', () => {
```

Comment:
The `cheerio` related tests should probably be in the `views` directory, not the `handlers` directory, as they're testing the njk files rather than the ts handlers

Reply:
That file was existing so i used the same to test my scenarios

Reply:
Was the existing file not just checking the handler rather than the view?

Reply:
I don't think these specs need to be here anymore do they? They appear to be in a new spec called `viewActive`

Reply:
Yes, removed.